tokens/properties/components/Checkbox.json

```diff
@@ -0,0 +1,127 @@
+{
+    "wikit-Checkbox": {
+        "label": {
+            "font-family": {
+                "value": "{font.family.style.label.value}"
+            },
+            "font-weight": {
+                "value": "{font.weight.style.label.value}"
+            },
+            "font-size": {
+                "value": "{font.size.style.label.value}"
+            },
+            "color": {
+                "value": "{font.color.base.value}"
+            },
+            "color-disabled": {
+                "value": "{font.color.disabled.value}"
+            },
+            "line-height": {
+                "value": "{font.line-height.style.label.value}"
+            }
+        },
+        "control": {
+            "border-width": {
+                "value": "{border.width.thin.value}"
+            },
+            "border-style": {
+                "value": "{border.style.base.value}"
+            },
+            "border-radius": {
+                "value": "{border.radius.base.value}"
+            },
+            "background-color": {
+                "value": "{background.color.base.default.value}"
+            },
+            "height": {
+                "value": "{dimension.min-height.small.value}"
+            },
+            "width": {
+                "value": "{dimension.width.small.value}"
+            },
+            "right-spacing": {
+                "value": "{dimension.spacing.small.value}"
+            },
+            "transition-timing-function": {
+                "value": "{transition.timing-function.ease.value}"
+            },
+            "transition-property": {
+                "value": "{transition.interaction.binary.property.value}"
+            },
+            "transition-duration": {
+                "value": "{transition.interaction.binary.duration.value}"
+            },
+            "unchecked": {
```

Comment:
hey, @tzhelyazkova. Checking [Termbox's implementation of the checkbox](https://github.com/wikimedia/wikibase-termbox/blob/master/src/components/Checkbox.vue) made me think that having this "unchecked" tokens might be redundant, since we might not need this pseudo-class?  What do you think?

Reply:
I agree. :+1: Anything that defines the default state of the box should be under "control".
Anything that changes style-wise when the box is checked should go under "checked".